torch/tensor.py

```diff
@@ -818,15 +818,20 @@ def unflatten(self, dim, sizes):
         Examples:
             >>> torch.randn(3, 4, 1).unflatten(1, (2, 2)).shape
             torch.Size([3, 2, 2, 1])
+            >>> torch.randn(3, 4, 1).unflatten(1, (-1, 2)).shape # the size -1 is inferred from other dimensions
```

Comment:
Nit: -1 isn't really inferred from "the other dimensions", -1 is inferred from the size of dimension 1

Reply:
Addressed.